spec/api/sul_bib/authorship_api_spec.rb

```diff
@@ -24,57 +18,24 @@
     pub.save # to update the pub.pub_hash
     pub
   end
-  let(:valid_data_for_post) do
+  let(:base_data) do
     {
-      sul_pub_id: publication.id,
       featured: false,
       status: 'denied',
       visibility: 'private',
     }
   end
-  let(:new_authorship_for_pub_with_contributions) do
-    # status 'approved', visibility 'public', featured true
-    # In this submission, use different values.
-    {
-      sul_pub_id: publication_with_contributions.id,
-      featured: false,
-      status: 'denied',
-      visibility: 'private'
-    }
-  end
-  let(:authorship_for_sw_id) do
-    {
-      sw_id: '10379039',
-      featured: false,
-      status: 'denied',
-      visibility: 'private'
-    }
-  end
-  let(:authorship_for_pmid) do
-    {
-      pmid: '23684686',
-      featured: false,
-      status: 'denied',
-      visibility: 'private'
-    }
-  end
-  let(:existing_contrib) do
-    publication_with_contributions.contributions.first
-  end
+  let(:valid_data_for_post) { base_data.merge(sul_pub_id: publication.id) }
+  let(:new_authorship_for_pub_with_contributions) { base_data.merge(sul_pub_id: publication_with_contributions.id) }
+  let(:existing_contrib) { publication_with_contributions.contributions.first }
```

Comment:
The reason all these data are grouped up here at the top of the spec is so it's easy to see what is different between them.  Looks like `authorship_for_sw_id` and `authorship_for_pmid`  moved somewhere else.  It's not a blocker, just noting that when they are all together it's easier to see what's different.